polymer.externs.js

```diff
@@ -108,12 +108,11 @@ PolymerElement.prototype.setPathValue = function(path, value) {};
  *
  * @param {string} type An event name.
  * @param {*=} detail
- * @param {Node=} onNode Target node.
- * @param {boolean=} bubbles Set false to prevent bubbling, defaults to true.
- * @param {boolean=} cancelable Set false to prevent cancellation, defaults to
- *     true.
+ * @param {{
+ *   bubbles: (boolean|undefined),
+ *   cancelable: (boolean|undefined),
+ *   node: (!HTMLElement|undefined)}=} options
```

Comment:
Pretty sure that's what I'm doing here. See something I'm missing? I'll change the *= on detail to Object=.